src/Renderer/Camera.js

```diff
@@ -107,4 +107,19 @@ Camera.prototype.box3DSizeOnScreen = function box3DSizeOnScreen(box3d, matrixWor
     return c.applyMatrix4(m);
 };
 
+Camera.prototype.isSphereVisible = function isSphereVisible(sphere, matrixWorld) {
```

Comment:
Maybe you could extract the shared code between this function and isBox3dVisible to a private function ?